.goreleaser.yml

```diff
@@ -14,10 +15,12 @@ builds:
     binary: umeed
     mod_timestamp: "{{ .CommitTimestamp }}"
     flags:
-      - -tags=badgerdb ledger netgo
+      - -tags=badgerdb ledger netgo muslc
```

Comment:
Why adding muslc? We release glibc compatible binaires.

Reply:
According to this [comment](https://github.com/CosmWasm/wasmd/issues/268#issuecomment-826358561) it's not possible to build a statically linked binary for glibc systems. Using `muslc` seems to be the only way to create statically linked deterministic binaries.